doc/client_design.md

```diff
@@ -39,8 +39,44 @@ current-datacenter: datacenter1
 
 # Client commands
 
-## Reference
+## PaddleCloud Server command-line design
+You can use the following syntax to run `pcloud` command-line interface tool from your terminal window:
+`pcloud [sub-command] [resource] [name] [flags]`
+  - `sub-command`: The operation that you want to perform on the resource, for example, `submit`, `get`, `kill`, `describe`.
+  - `resource`: The resource types for Paddle job, for example `job`, `worker`, `quota`.
+  - `name`: You can specify a name for the given resource
+  - `flags`: You can specify any flags override default values for `command` or `resource`
+
+- Sub-Command
+  - `submit`: submit the specify resource, we only support submit a PaddleJob for now
+  - `get`: get all instances for the specified resource
+  - `describe`: output the description for the specify resource
+  - `kill`: kill the specific job 
+  - `logs`: output the logs for the specific instance, if the resource is a job, output all the workers' logs.
+
+- Resource
+  - `job`, PaddleJob running on PaddleCloud
+  - `workers`, The workers for PaddleJob, for example the master, trainer or pserver.
+  - `quota`, The quota for the current user
+
+- Example
+  - `pcloud submit job -name <job-name> -cpu 1 ...`
+  - `pcloud get [job|worker|quota]`
```

Comment:
If we want to separate `trainer` and `pserver`, we also need `master`, I think using **workers** to express `pserver`, `trainer` and `master` is good for simply users' operation, users can use a single command to view all the workers state: 
```bash
pcloud workers ...
```
rather than three commands:
```bash
pcloud trainers ...
pcloud servers ...
pcloud master ...
```